src/EnergyPlus/Furnaces.cc

```diff
@@ -6540,7 +6525,7 @@ namespace Furnaces {
         int FurnaceOutletNode = thisFurnace.FurnaceOutletNodeNum;
         int FurnaceInletNode = thisFurnace.FurnaceInletNodeNum;
         int ControlZoneNode = thisFurnace.NodeNumOfControlledZone;
-        int OpMode = thisFurnace.OpMode; // fan operating mode
+        HVAC::FanOp fanOp = thisFurnace.fanOp; // fan operating mode
```

Comment:
I just want to point out that `OpMode` is a bad variable name. `OpMode` of what?